mailpoet/assets/js/src/common/listings/newsletter_status.tsx

```diff
@@ -28,7 +28,7 @@ function NewsletterStatus({
   const sent = (!scheduledFor || isPast(scheduledFor)) && processed >= total;
   const sentWithoutQueue = status === 'sent' && total === undefined;
   let percentage = 0;
-  let label = t('notSentYet');
+  let label : string|JSX.Element = t('notSentYet');
```

Comment:
Doesn't `JSX.Element` already include `string`?

Reply:
[Type 'string' is not assignable to type 'Element'.](https://www.typescriptlang.org/play?#code/FAYw9gdgzgLgBAIwIYCcBcApAygDQHQCiANgKYC2JE8AvHAOQwmx0DcQA)

Reply:
👍